L1Trigger/CSCTriggerPrimitives/src/CSCUpgradeMotherboard.cc

```diff
@@ -0,0 +1,103 @@
+#include "L1Trigger/CSCTriggerPrimitives/src/CSCUpgradeMotherboard.h"
+#include "DataFormats/MuonDetId/interface/CSCTriggerNumbering.h"
+
+std::vector<CSCCorrelatedLCTDigi> CSCUpgradeMotherboard::LCTContainer::getTimeMatched(const int bx) const 
```

Comment:
better to avoid the expensive copy of the return type by making it an argument (eg, std::vector<CSCCorrelatedLCGDigi>&lcts)